presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -193,6 +193,16 @@ public Response postStatement(
         return Response.ok(query.getQueryResults(query.getLastToken(), uriInfo)).build();
     }
 
+    /**
+     * Retrieves the current status for a queued query.
```

Comment:
I think it's a little strange to use "Retrieves" on a void method. Do you think we can improve it anyway?

Reply:
improved